lib/services/platform-service.ts

```diff
@@ -187,6 +192,10 @@ export class PlatformService implements IPlatformService {
 				this.$pluginsService.ensureAllDependenciesAreInstalled().wait();
 				let tnsModulesDestinationPath = path.join(platformData.appDestinationDirectoryPath, constants.APP_FOLDER_NAME, PlatformService.TNS_MODULES_FOLDER_NAME);
 				this.$broccoliBuilder.prepareNodeModules(tnsModulesDestinationPath, this.$projectData.projectDir, platform, lastModifiedTime).wait();
+
+				if (this.$options.release) {
+					shell.rm("-rf", path.join(tnsModulesDestinationPath, PlatformService.UNIT_TEST_RUNNER_MODULE_NAME));
```

Comment:
What happens after I build for release once? Can I again run unit tests without doing anything special?